docs/deployment/deploy_gardenlet_manually.md

```diff
@@ -0,0 +1,402 @@
+
+Once you’ve deployed the initial gardenlet manually, you can deploy new gardenlets automatically using the initial gardenlet as a template.
+
+More information: [Automatic Deployment of Gardenlets](deploy_gardenlet_automatically.md).
+
+## Prerequisites
+
+### Kubernetes cluster that should be registered as a seed cluster
+    
+- Verify that the cluster has a [supported Kubernetes version](../usage/supported_k8s_versions.md).
+- Determine the nodes, pods, and services CIDR of the cluster. You need to configure this information in the `Seed` configuration. Gardener uses this information to check that the shoot cluster isn’t created with overlapping CIDR ranges.
+- You need to have a predeployed `Ingress` controller. The ingress controller is preinstalled for shooted seed clusters. An ingress controller is required to:
+
+  - configure the `Seed` cluster resource (tells the gardenlet the ingress domain to use to deploy ingress resources for seed components like grafana and prometheus).
+      
+  - handle ingress resources that are deployed during the `Seed` bootstrap by the gardenlet.
+      
+  >:warning:<br>  
+  There must be a DNS record `*.ingress.<SEED-CLUSTER-DOMAIN>` where `<SEED-CLUSTER-DOMAIN>` is the value of the `.dns.ingressDomain` field of [a seed cluster resource](../../example/50-seed.yaml) (or configured using the `seedConfig{}` in the [gardenlet configuration](../../example/20-componentconfig-gardenlet.yaml).
+  >
+  >Here is an example for the Nginx ingress controller:
+  >
+  > 1. Deploy Nginx into the `kube-system` namespace in the Kubernetes cluster that you want to be registered as a `Seed`.
+  >
+  >    On most cloud providers, Nginx creates the service with type `LoadBalancer` with an external IP. 
+  >
+  >    ```
+  >    NAME                        TYPE           CLUSTER-IP    EXTERNAL-IP
+  >    nginx-ingress-controller    LoadBalancer   10.0.15.46    34.200.30.30
+  >    ```
+  >
+  > 1. Create a wildcard `A` record (for example, `*.ingress.sweet-seed.<my-domain>. IN A 34.200.30.30`) with your DNS provider and point it to the external IP of the ingress service. This ingress domain is later required to register the `Seed` cluster.
+
+
+### `kubeconfig` For the Seed Cluster
+
+The `kubeconfig` is required to deploy the gardenlet Helm chart to the seed cluster. This deployment requires admin privileges. The Helm chart contains a service account `gardenlet` that the gardenlet deployment uses by default to talk to the Seed API server.
+
+> If the gardenlet isn’t deployed in the seed cluster, the gardenlet can be configured to use a `kubeconfig`, which also requires full admin rights, from a mounted directory. The `kubeconfig` is shared in section `seedClientConnection.kubeconfig` of the [Gardenlet configuration](../../example/20-componentconfig-gardenlet.yaml). 
+As this document describes the Helm-based setup on the seed cluster, this configuration option isn’t used in the following. 

```

Comment:
```suggestion
This configuration option isn’t used in  the following,  as this document only describes the recommend setup option where the Gardenlet is running in the Seed cluster itself. 
```

Reply:
The focus is on "Gardenlet is running in the Seed cluster itself"  not because it is deployed by helm.

Reply:
done